```
cmake -B build -DCMAKE_BUILD_TYPE=Release
Version: 0.9.10
-- The C compiler identification is AppleClang 16.0.0.16000026
-- The CXX compiler identification is AppleClang 16.0.0.16000026
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PkgConfig: /opt/homebrew/bin/pkg-config (found version "2.5.1")
-- Checking for module 'erfa>=1.7'
--   Found erfa, version 2.0.1
-- Configuring done (0.6s)
-- Generating done (0.0s)
-- Build files have been written to: /Users/timj/work/pal/build
```

```
cmake --build build
[  1%] Building C object CMakeFiles/pal.dir/pal1Atms.c.o
[  2%] Building C object CMakeFiles/pal.dir/pal1Atmt.c.o
[  3%] Building C object CMakeFiles/pal.dir/palAddet.c.o
In file included from /Users/timj/work/pal/palAddet.c:86:
/Users/timj/work/pal/pal1sofa.h:136:12: fatal error: 'erfa.h' file not found
  136 | #  include "erfa.h"
      |            ^~~~~~~~
1 error generated.
make[2]: *** [CMakeFiles/pal.dir/palAddet.c.o] Error 1
make[1]: *** [CMakeFiles/pal.dir/all] Error 2
make: *** [all] Error 2
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,0 +1,39 @@
+# cspell:disable
+cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
+
+set(VER_MAJ 0) # major
+set(VER_MIN 9) # minor
+set(VER_PAT 10) # patch
+set(VER_SS ${VER_MAJ}.${VER_MIN}.${VER_PAT})
+message("Version: ${VER_SS}")
+
+project(cx_pal VERSION ${VER_SS})
+
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+add_definitions(-DPACKAGE_VERSION="${VER_SS}")
+add_definitions(-DPACKAGE_VERSION_INTEGER=${VER_MAJ})
+
+set(CMAKE_BUILD_TYPE Release)
+
+find_package(PkgConfig)
+pkg_check_modules(Erfa REQUIRED IMPORTED_TARGET erfa>=1.7)
+message(STATUS "Erfa Include Dirs: ${Erfa_INCLUDE_DIRS}")
+message(STATUS "Erfa Libraries: ${Erfa_LIBRARIES}")
+
+include_directories(${PROJECT_SOURCE_DIR}/ ${Erfa_INCLUDE_DIRS})
+
+file(GLOB sources ${PROJECT_SOURCE_DIR}/*.c)
+file(GLOB headers ${PROJECT_SOURCE_DIR}/*.h)
+
+set(LINK_LIBS PkgConfig::Erfa)
+
+add_library(pal SHARED ${sources} ${headers})
+target_link_libraries(pal ${LINK_LIBS})
+set_target_properties(pal PROPERTIES VERSION ${VER_SS} SOVERSION
+                                                          ${VER_MAJ})
+install(FILES ${headers} DESTINATION include/star)
+install(
+  TARGETS pal
+  RUNTIME DESTINATION bin/
+  LIBRARY DESTINATION lib/
+  ARCHIVE DESTINATION lib/)

